compliance_checker/suite.py

```diff
@@ -186,6 +186,7 @@ def _get_valid_checkers(self, ds, checker_names):
         the user selected names.
         """
 
+        import pdb; pdb.set_trace()
```

Comment:
Will need to be removed.

Reply:
@daltonkell, I sometimes add a client-side Git pre-commit hook to ensure there are no `pdb` statements present in the code I'm committing.  There are a number of example pre-commit hook scripts out in the wild that might be worth looking at.